### DISCUSSION TITLE
Content: HTML fields to always have visible selector

### DISCUSSION THREAD

**Discussion by vonscriptor** [solspace/craft-freeform#1184] [Feature Requests]
HTML fields in the Layout are hard or locate and select. Say you create an <hr style="margin: 0;"> there is 1px available to select it and almost no visual indication that it exists other than whitespace.

Would be nice to have a label and [dotted] outline around HTML fields and the like to help with accessibility.

Additionally, having a legend with used fields and their hierarchy allowing for selection, reordering, removal, etc could be a clear way of managing forms complimentary to the layout.

**Comment by kjmartens** [comment]
Thanks for your feedback @vonscriptor. I think that's a fair observation. We'll likely add this in shortly. 🙂

**Comment by kjmartens** [comment]
This should be resolved in Freeform 5.1.11 now. 🙂
